### PR TITLE
feat(tapable): convert JsonpMainTemplatePlugin to use .tap

### PR DESCRIPTION
**Pull request by TheLarkInn**
**What kind of change does this PR introduce?**
Refactoring 

**Did you add tests for your changes?**
N/A

**If relevant, link to documentation update:**
N/A

**Summary**
Update the calls in JsonpMainTemplatePlugin to use `.tap` instead of `.plugin`

**Does this PR introduce a breaking change?**
Nope. 

**Other information**

### CODE REVIEW

File: lib/web/JsonpMainTemplatePlugin.js [hunk 1]
```diff
@@ -16,9 +16,11 @@ class JsonpMainTemplatePlugin {
 			return otherChunksInEntry || onDemandChunks;
 		};
 		// TODO refactor this
-		if(!mainTemplate.hooks.jsonpScript)
+		if(!mainTemplate.hooks.jsonpScript) {
 			mainTemplate.hooks.jsonpScript = new SyncWaterfallHook(["source", "chunk", "hash"]);
-		mainTemplate.plugin("local-vars", (source, chunk) => {
+		}
+
+		mainTemplate.hooks.localVar.tap("JsonpMainTemplatePlugin", (source, chunk) => {
```
Comment (ooflorent) [inline lib/web/JsonpMainTemplatePlugin.js]: It's `localVars`

Comment (TheLarkInn) [reply lib/web/JsonpMainTemplatePlugin.js]: Thanks!

**Comment by ooflorent** [comment]
Another random node v6 failure...

**Comment by TheLarkInn** [comment]
I'll kick another cycle off and we'll see.

**Comment by sokra** [comment]
Thanks

**Pull request merged**